data/kubevirt-ipam-controller/004-primary-udn-networkattachdef.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: primary-user-defined-network
+  namespace: default
+spec:
+  config: '{
+  "cniVersion": "1.0.0",
+  "name": "primary-user-defined-network",
```

Comment:
Let's differenciate from nad name, since those are different 
```yaml
name: [CNAO namespace]-kubevirt-passt-binding-plugin-net
```

Reply:
After some back and forth let's name this 
```yaml
name: primary-user-defined-network-kubevirt-binding
```

Reply:
DONE